frontend/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "frontend",
   "version": "0.1.0",
   "private": true,
-  "homepage": "https://BitcoinDevShop.github.io/mutiny-web-poc",
+  "homepage": "https://BitcoinDevShop.github.io",
```

Comment:
I think this might screw up the github pages deployment, probably fine to merge and see what happens

Reply:
I don't think so, it seems to be triggered by that `homepage` url.

Reply:
Agree just to see what happens